### PR TITLE
#qgAq5WKe Log out  Authenticated User

### PR DESCRIPTION
**Pull request by Mugema123**
**What does this PR do?**
-Creating a log out system for an authenticated user.

**Description of Task to be completed?**
 -To enable the users to log out  by destroying token.
 -The Authenticated user must access the protected resources

**how should this be manually tested**
clone the repo
run npm install
run npm run dev
run http:localhost:2020/login in postman
run http:localhost:2020/user in postman
run http:localhost:2020/logout in postman
Any background context you want to provide?

**What are the relevant pivotal tracker stories?**
#qgAq5WKe

**Screenshots**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -16,14 +17,16 @@ app.use(express.json());
...
+
 
 app.use(translator);
 
 app.use(cors());
 
 app.use('/api-docs', swaggerUI.serve, swaggerUI.setup());
 
-const port = process.env.PORT || 3020;
+const port = process.env.PORT || 2020;
```
Comment (Ntwali-Josue) [inline src/app.js]: why did you change the usual port ,we were using?

**Review by Ntwali-Josue — COMMENTED** [review]
Try to have one commit

File: src/app.js (hunk 1 above)
Comment (Mugema123) [reply src/app.js]: No,I did not , check yours too

Comment (Ntwali-Josue) [reply src/app.js]: for us we're using 3020,and for you we're supposed to use 2020!?
